build/build.rb

```diff
@@ -89,6 +89,10 @@ def verify_run(output)
     file_name = "#{name}-#{target}-#{build_label}-#{timestamp}-#{git_checkout.commit_sha}.#{FILE_TYPE[imgfac_target]}"
     destination = DESTINATION_DIRECTORY.join(file_name)
     $log.info `mv  #{source} #{destination}`
+    $log.info "Generating image checksums"
```

Comment:
@JPrause Sorry but I must be missing something. My suggest is limited only to suggestions for the changes you are making for this one PR. I only see 3 lines that use the "cd #{destination_directory}; ..." and they all were added by you for this PR. 

My suggestion is to research leveraging Dir.chdir in an attempt to avoid having to issue the "cd #{destination_directory}" to the commands you are adding.

Let's connect to discuss because it seems we have a disconnect.